examples/acs-kubernetes/terraform.tfvars

```diff
@@ -0,0 +1,19 @@
+resource_group_name = "AzureQuickstart"
+
+resource_group_location = "West US"
+
+dns_name_prefix = "REPLACE_WITH_UNIQUE_NAME"
+
+linux_agent_count = "3"
+
+linux_agent_vm_size = "Standard_D2_v2"
+
+linux_admin_username = "azure"
+
+linux_admin_ssh_publickey = "REPLACE_WITH_SSHKEY"
```

Comment:
🤔 would it make sense for this to be the path to the public key (defaulted to `~/.ssh/id_rsa.pub`?) and to load that, rather than having the user input it?